adapter_pipelines/ss2_single_end/adapter.wdl

```diff
@@ -0,0 +1,207 @@
+import "SmartSeq2SingleSampleUnpaired.wdl" as ss2
+import "submit.wdl" as submit_wdl
+
+
+task GetInputs {
+  String bundle_uuid
+  String bundle_version
+  String dss_url
+  Int? retry_timeout
+  Float? retry_multiplier
+  Int? retry_max_interval
+  Int? individual_request_timeout
+  Boolean record_http
+  String pipeline_tools_version
+
+  command <<<
+    export RECORD_HTTP_REQUESTS="${record_http}"
+    export RETRY_TIMEOUT="${retry_timeout}"
+    export RETRY_MULTIPLIER="${retry_multiplier}"
+    export RETRY_MAX_INTERVAL="${retry_max_interval}"
+    export INDIVIDUAL_REQUEST_TIMEOUT="${individual_request_timeout}"
+
+    # Force the binary layer of the stdout and stderr streams to be unbuffered.
+    python -u <<CODE
+    from pipeline_tools.pipelines.smartseq2 import smartseq2
+
+    smartseq2.create_ss2_se_input_tsv(
+                  "${bundle_uuid}",
+                  "${bundle_version}",
+                  "${dss_url}")
+
+    CODE
+  >>>
+  runtime {
+    docker: "quay.io/humancellatlas/secondary-analysis-pipeline-tools:" + pipeline_tools_version
+  }
+  output {
+    Array[File] http_requests = glob("request_*.txt")
+    Array[File] http_responses = glob("response_*.txt")
+    Object inputs = read_object("inputs.tsv")
+  }
+}
+
+workflow AdapterSmartSeq2SingleCellUnpaired {
+  # variable parameters provided by the notification sent by the data storage service
+  String bundle_uuid
+  String bundle_version
+
+  # fixed parameters
+  File genome_ref_fasta
+  File rrna_intervals
+  File gene_ref_flat
+  File hisat2_ref_index
+  File hisat2_ref_trans_index
+  File rsem_ref_index
+  File hisat2_ref_name
+  File hisat2_ref_trans_name
+  String stranded
+
+  # submission parameters
+  File format_map
+  String dss_url
+  String submit_url
+  String method
+  String schema_url
+  String cromwell_url
+  String analysis_process_schema_version
+  String analysis_protocol_schema_version
+  String analysis_file_version
+  String run_type
+  Int? retry_max_interval
+  Float? retry_multiplier
+  Int? retry_timeout
+  Int? individual_request_timeout
+  String reference_bundle
+
+  # Set runtime environment such as "dev" or "staging" or "prod" so submit task could choose proper docker image to use
+  String runtime_environment
+  # By default, don't record http requests, unless we override in inputs json
+  Boolean record_http = false
+  Int max_cromwell_retries = 0
+  Boolean add_md5s = false
+
+  String pipeline_tools_version = "v0.50.2"
```

Comment:
It's annoying but until we separate the adapter WDLs from this repo, anything that changes the `pipeline-tools` package requires you to change this to a "pre-release version" for all of the WDLs. For instance, in this case, you'd change bump this to `v0.51.0` (even it's not tagged yet), and also change this line in all of the existing adapter WDLs.

Reply:
Fixed